scripts/package/package-debian.sh

```diff
@@ -17,46 +17,74 @@ done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
 source "$DIR/../common/_common.sh"
-
-if [ "$OSNAME" != "ubuntu" ]; then
-    error "Debian Package build only supported on Ubuntu"
+REPOROOT="$DIR/../.."
+
+help(){
+    echo "Usage: $0"
+    echo ""
+    echo "Options:"
+    echo "  --version <version>                Specify a version for the package."
+    echo "  --input <input directory>          Package the entire contents of the directory tree."
```

Comment:
The input is just the stage2 directory. No? https://github.com/dotnet/cli/pull/1739/files#diff-849277af96dad943db69a538f55f0a97R50